File: api/v2010/message/create-example-3/create-example-3.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

message = @client.messages
                 .create(
                    body: "Let's grab lunch at Milliways tomorrow!",
                    from: '+14158141829',
                    media_url: 'http://www.example.com/cheeseburger.png',
                    to: '+15558675310'
                  )

puts message.sid

File: api/v2010/message/create-example-3/create-example-3.5.x.cs
// Install the C# / .NET helper library from twilio.com/docs/csharp/install

using System;
using System.Linq;
using Twilio;
using Twilio.Rest.Api.V2010.Account;


class Program 
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Token at twilio.com/console
        // DANGER! This is insecure. See http://twil.io/secure
        const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        const string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var mediaUrl = new [] {
            new Uri("http://www.example.com/cheeseburger.png")
        }.ToList();

        var message = MessageResource.Create(
            body: "Let's grab lunch at Milliways tomorrow!",
            from: new Twilio.Types.PhoneNumber("+14158141829"),
            mediaUrl: mediaUrl,
            to: new Twilio.Types.PhoneNumber("+15558675310")
        );

        Console.WriteLine(message.Sid);
    }
}
